app/assets/stylesheets/shared/tables.scss

```diff
@@ -114,23 +90,15 @@
   text-decoration: none;
   font-size: 1.1em;
   line-height: 1.5;
-  border-radius: 6px;;
-  font-weight: 700;;
-  background: $medium-gray;;
-  color: #fff;;
+  border-radius: 6px;
+  font-weight: 700;
+  background: $medium-gray;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#fff` should be written as `#FFF`